sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs

```diff
@@ -169,16 +169,17 @@ public static OpenTelemetryBuilder UseAzureMonitor(this OpenTelemetryBuilder bui
                             AzureMonitorAspNetCoreEventSource.Log.GetEnvironmentVariableFailed(EnableLogSamplingEnvVar, ex);
                         }
 
-                        if (enableLogSampling)
-                        {
-                            var azureMonitorExporterOptions = new AzureMonitorExporterOptions();
-                            azureMonitorOptions.SetValueToExporterOptions(azureMonitorExporterOptions);
-                            loggingOptions.AddProcessor(new LogFilteringProcessor(new AzureMonitorLogExporter(azureMonitorExporterOptions)));
-                        }
-                        else
-                        {
-                            loggingOptions.AddAzureMonitorLogExporter(o => azureMonitorOptions.SetValueToExporterOptions(o));
-                        }
+                        // TODO: WILL RE-ENABLE IN NEXT BETA
+                        //if (enableLogSampling)
+                        //{
+                        //    var azureMonitorExporterOptions = new AzureMonitorExporterOptions();
+                        //    azureMonitorOptions.SetValueToExporterOptions(azureMonitorExporterOptions);
+                        //    loggingOptions.AddProcessor(new LogFilteringProcessor(new AzureMonitorLogExporter(azureMonitorExporterOptions)));
+                        //}
+                        //else
+                        //{
+                        //    loggingOptions.AddAzureMonitorLogExporter(o => azureMonitorOptions.SetValueToExporterOptions(o));
```

Comment:
how does distro export logs with these commented out?

Reply:
Sorry, i see my mistake. Trying to undo this change: https://github.com/Azure/azure-sdk-for-net/pull/41665/files

Reply:
fixed this. tested and confirmed that Logs are still exported:

![image](https://github.com/Azure/azure-sdk-for-net/assets/28785781/27c033f9-9d4a-4a74-b668-2cce20e453a1)